apps/site/assets/css/_schedule-page.scss

```diff
@@ -1,3 +1,28 @@
+@include media-breakpoint-down(sm) {
+  .m-schedule-page {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
This is a really cool use of the `order` attribute to get the result we want on mobile.
Since you're using flexbox I'm wondering if we can simplify the template a bit by removing some of the bootstrap wrappers. 

Currently the page is like this:

```html
<div class="page-section m-schedule-line">
    <div class="row">
        <div class="col-md-12">
           <ul class="c-alert-group">alerts</ul>
        </div>
        <div class="m-schedule-page">
            <div class="col-xs-12 col-md-7 m-schedule-page__main-content"></div>
            <div class="col-xs-12 col-md-5 col-lg-4 col-lg-offset-1 m-schedule-page__schedule-finder-or-note"></div>
            <div class="col-xs-12 col-md-5 col-lg-4 col-lg-offset-1 m-schedule-page__side-content"></div>
        </div>
    </div>
</div>
```

This should look the same and be a lot cleaner:

```html
<div class="page-section m-schedule-line">
    <ul class="c-alert-group">alerts</ul>
    <div class="m-schedule-page row">
        <div class="col-md-7 m-schedule-page__main-content"></div>
        <div class="col-md-5 col-lg-4 col-lg-offset-1 m-schedule-page__schedule-finder-or-note"></div>
        <div class="col-md-5 col-lg-4 col-lg-offset-1 m-schedule-page__side-content"></div>
    </div>
</div>
```

* Since the alerts are always full width we don't really need to manage its layout with any special classes
* We can move the `.row` class to directly wrap the rest of the `.col-` elements
* We can remove `.col-xs-12` generally... this is the default behavior! :)

Reply:
Wow that is a lot simpler indeed...the latest changes should contain your suggested changes!